ui/admin/tests/e2e/README.md

```diff
@@ -136,6 +136,11 @@ PWDEBUG=console yarn playwright test --headed --config ./tests/e2e/playwright.co
 PWDEBUG=console yarn playwright test --headed --config ./tests/e2e/playwright.config.js login.spec.js --debug
 ```
 
+To run the tests with a specific tag (e.g. @enterprise), use these commands.
+```bash
+PWDEBUG=console yarn playwright test --grep @enterprise --headed --config ./tests/e2e/playwright.config.js delete-resources.spec.js
+```
```

Comment:
Would would be our use cases be? I see two major ones:
- run all tests applicable to Community Edition
- run all tests applicable to Enterprise (which may include tests from the first set)

How would we be able to achieve those?

Reply:
Yeah, in these cases we'd do:

**Community Edition tests:**
`PWDEBUG=console yarn playwright test --grep-invert @enterprise --headed --config ./tests/e2e/playwright.config.js delete-resources.spec.js
`
That will run all tests except those tagged @hashicorp/team-grc-compliance 

**Enterprise tests (i.e. all tests)**
`PWDEBUG=console yarn playwright test --headed --config ./tests/e2e/playwright.config.js delete-resources.spec.js`
That will run all tests.